apps/cli/src/node/docker-compose-dev.yml

```diff
@@ -90,6 +90,8 @@ services:
             retries: 5
         ports:
             - 8080:8080
+        env_file:
+            - ./.sunodo.env
```

Comment:
Does this should be here? I see you are adding a compose file if the env file exists.

Reply:
Fixed!